internal/condition/deployment_config_condition.go

```diff
@@ -1,17 +1,18 @@
 package condition
 
 import (
-	"errors"
 	"fmt"
-	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 	"time"
+
+	"github.com/fabric8-services/fabric8-jenkins-idler/internal/model"
 )
 
-// DCCondition covers changes to Jenkins DeploymentConfigs
+// DeploymentConfigCondition covers changes to Jenkins DeploymentConfigs
```

Comment:
that is not correct per se. It does not have to be Jenkins. It is a DeploymentConfiguration in general.